File: frontend/lib/models/plan/edit_treasurer_response.dart
// lib/models/plan/edit_treasurer_response.dart
class EditTreasurerResponse {
  final int planId;
  final int oldTreasurerId;
  final String oldTreasurerNickname;
  final int newTreasurerId;
  final String newTreasurerNickname;

  EditTreasurerResponse({
    required this.planId,
    required this.oldTreasurerId,
    required this.oldTreasurerNickname,
    required this.newTreasurerId,
    required this.newTreasurerNickname,
  });

  factory EditTreasurerResponse.fromJson(Map<String, dynamic> json) {
    return EditTreasurerResponse(
      planId: json['planId'],
      oldTreasurerId: json['oldTreasurerId'],
      oldTreasurerNickname: json['oldTreasurerNickname'],
      newTreasurerId: json['newTreasurerId'],
      newTreasurerNickname: json['newTreasurerNickname'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'planId': planId,
      'oldTreasurerId': oldTreasurerId,
      'oldTreasurerNickname': oldTreasurerNickname,
      'newTreasurerId': newTreasurerId,
      'newTreasurerNickname': newTreasurerNickname,
    };
  }
}

File: backend/src/main/java/com/patriot/fourlipsclover/plan/dto/response/EditTreasurerResponse.java
package com.patriot.fourlipsclover.plan.dto.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class EditTreasurerResponse {

    private int planId;

    private Long oldTreasurerId;
    private String oldTreasurerNickname;

    private Long newTreasurerId;
    private String newTreasurerNickname;

}
